data/data/azure/dns/dns.tf

```diff
@@ -50,6 +50,15 @@ resource "azureprivatedns_a_record" "etcd_a_nodes" {
   records             = [var.etcd_ip_addresses[count.index]]
 }
 
+source "azurerm_dns_a_record" "etcd_a_bootstrap" {
+  count               = 1
+  name                = "bootstrap"
```

Comment:
`https://github.com/openshift/installer/pull/2608/files#diff-1dd6be44797de18d5c235fc324f8ee2aR92`

i think this should be something like https://github.com/openshift/installer/pull/2608/files#diff-764adb23dcc0edbbebc09192eb233e9aR3